capstone/src/main/java/com/google/sps/servlets/Club.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.servlets;
+
+import java.util.List;
+
+/* Contains all information relevant to a club. */
+public class Club {
+  private String name;
+  private List<String> members;
```

Comment:
It's good practice to make your fields immutable when possible. So let's make this List an ImmutableList, same with officers and announcements below. 

If you ever need to add or remove an object from the list, you can do that with individual getter/setter implementations, depending on the use case.